src/redux/actions/authActions.js

```diff
@@ -0,0 +1,39 @@
+import {createAsyncThunk, createAction} from '@reduxjs/toolkit';
+
+import client from '../../api/apiInstance';
+import {basicHeaders, noTokenHeaders} from '../../config/apiConfig';
+
+export const login = createAsyncThunk("auth/login", async (data, thunkAPI) => {
+  return client.api.accountLoginCreate(data, {headers: noTokenHeaders()})
+    .then((response) => response.data)
+    .catch((response) => {
+      if (response.error) {
+        response.error.status = response.status;
+      }
+
+      throw thunkAPI.rejectWithValue(response.error)
+    });
+});
+
+export const validateLogin = createAsyncThunk("auth/validateLogin", async (data, thunkAPI) => {
+  return client.api.accountIsTokenValidList({ headers: basicHeaders() })
+    .then((response) => {
+      console.log('Valid token');
+
+      return response.data;
+    })
+    .catch((response) => {
+      if (response.error) {
+        response.error.status = response.status;
+      }
+
+      throw thunkAPI.rejectWithValue({
+        status: response.status,
+        // message: response.error.message,
```

Comment:
comment to be deleted

Reply:
I left it here because I'm not sure if we want to display an error message from the response or if we want to hardcode it.